ui-tests/src/test/java/org/openmrs/reference/page/HtmlFormsPage.java

```diff
@@ -0,0 +1,41 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class HtmlFormsPage extends Page {
+
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
+    private static final By NAME = By.name("form.name");
+    private static  By DESCRIPTION = By.name("form.description");
+    private static  By SAVE_FORM = By.cssSelector("input[type=\"submit\"]");
+    private  static By VERSION = By.name("form.version");
+
+    public HtmlFormsPage(ManageHtmlFormsPage page) {
+        super(page);
+    }
+
+   public void setNAME(String name) throws InterruptedException {
+        driver.findElement(NAME).clear();
+        driver.findElement(NAME).sendKeys("newFormTest");
+        driver.findElement(NAME).getAttribute("value");
+    }
+    public  void setDESCRIPTION(String description) {
+       driver.findElement(DESCRIPTION).clear();
+      driver.findElement(DESCRIPTION).sendKeys("description");
+      driver.findElement(DESCRIPTION).getAttribute("value");
+    }
+   public void setVERSION(String integer){
```

Comment:
these attribute setting methods can all be merged into one method such as createNewTestForm()

Reply:
Resolved thanks